### PR TITLE
Add msi-qe Service Account

### PR DESCRIPTION
**Pull request by chetna14manku**
Add MSI QE Service Account for pulling images from ci registry

### CODE REVIEW

**Review by rnetser — APPROVED** [review]

File: clusters/app.ci/registry-access/msi-qe/admin_manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  name: image-puller
+  namespace: msi-qe
+---
+# this grants your ServiceAccount rights to pull images
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: msi-qe-image-puller-binding
+  # the namespace from which you will pull images
+  namespace: ocp
```
Comment (smg247) [inline clusters/app.ci/registry-access/msi-qe/admin_manifest.yaml:23]: is this the correct ns?

**Pull request merged**